packages/checkbox-group/test/visual/material/checkbox-group.test.js

```diff
@@ -1,5 +1,6 @@
 import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
 import { visualDiff } from '@web/test-runner-visual-regression';
+import '@vaadin/checkbox/theme/material/vaadin-checkbox.js';
```

Comment:
Same question as above (see comment for Lumo).

Reply:
Fixed.